packages/botonic-react/src/msg-to-botonic.jsx

```diff
@@ -169,13 +169,15 @@ function buttonsParse(buttons) {
     const target = props.messenger_extensions ? null : props.target
     const title = props.title
     const webview = props.messenger_extensions ? props.url : props.webview
+    const enabled = props.enabled !== undefined ? props.enabled : true
     return (
       <Button
         key={i}
         payload={payload}
         url={url}
         target={target}
         webview={webview}
+        enabled={enabled}
```

Comment:
Should be `parentId` property added here?

Reply:
yes, sure, added 👍